docs/content/src/learn/guides/integration-patterns/scatter-gather-flow/README.md

```diff
@@ -0,0 +1,209 @@
+# Scatter-Gather Flow Control
+
+## About
+
+Scatter-Gather is an integration pattern where a request is sent to multiple recipients and each of the responses are aggregated and returned back to the client as a single response. This guide demonstrates a simple scatter-gather scenario where two files in an FTP location are read simultaneously, their contents aggregated and sent back to the client.
+
+## What you'll build
+
+We shall have a service ‘employeeDetails’ which accepts a client request and reads two different CSV files with employee details from an FTP server. It then converts the responses of each read to json, aggregates both the json and sends back to the client.
+
+## Prerequisites
+
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
+> **Tip**: For a better development experience, install the Ballerina IDE plugin for [VS Code](https://marketplace.visualstudio.com/items?itemName=ballerina.ballerina)
+- An FTP Server
+
+## Implementation
+
+* Create a new ballerina project named ‘scatter-gather-flow’.
```

Comment:
```suggestion
* Create a new Ballerina project named `scatter-gather-flow`.
```

Let's change "ballerina" to "Ballerina" everywhere. Also let's use backticks instead of single quotes for project names, file names, and anything related to the code.